src/incubator/TextField/FloatingPlaceholder.tsx

```diff
@@ -18,24 +19,27 @@ const FloatingPlaceholder = ({
   floatOnFocus,
   validationMessagePosition,
   extraOffset = 0,
+  defaultValue,
```

Comment:
The FlotingPlaceholder only needs to know only if to float or not, why doesn't the TextField component controls it? Like passing in 'shouldFloat' prop as boolean that will be updated by the state of the input's value?

Reply:
You're saying that `shouldFloat` should be coming from somewhere else? Maybe the `Presenter` like in `ValidationMessage`?

Reply:
Yes. I think the floating placeholder should only get a boolean and the logic will be in the component that uses it